internal/server/models/group.go

```diff
@@ -8,8 +8,9 @@ import (
 type Group struct {
 	Model
 
-	Name      string `gorm:"uniqueIndex:idx_groups_name,where:deleted_at is NULL"`
-	CreatedBy uid.ID
+	Name              string `gorm:"uniqueIndex:idx_groups_name,where:deleted_at is NULL"`
+	CreatedBy         uid.ID
```

Comment:
What does `CreatedBy` get set to when a provider creates the group?

Reply:
This is a good catch. I wasn't filling out `CreatedBy` at all, and it also doesn't get filled out when it gets created by the provider either. I think it probably makes the most sense to have two fields though; one specifically w/ the uid of the provider when it's automatically created, and the other one with the uid of the person who manually creates it.

Reply:
We could additionally fill in the `CreatedBy` field for the user who triggered a group to be automatically created by the provider, but I'm not certain how useful that is.

Reply:
I like your suggestion of two separate fields, one to track the up-stream provider the group came from and one for the account that created the group. Having the group "CreatedBy" whichever user first logs in with a given group seems confusing.